tests/integration/test_model.py

```diff
@@ -1036,3 +1054,128 @@ def test_predict_with_nullable_types(c):
         result,
         check_dtype=False,
     )
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 9),
+    reason="Some newer sklearn classes are only available with Python version >= 3.9",
+)
+@pytest.mark.parametrize("gpu", [False, pytest.param(True, marks=pytest.mark.gpu)])
+def test_ml_class_mappings(gpu):
```

Comment:
Since this is testing the mappings directly, maybe this would make sense in its own file in the unit tests? Or we could merge this into `test_ml_wrapppers.py` and just have that be something like `test_ml_utils.py`